test/Pool.test.js

```diff
@@ -1315,6 +1315,196 @@ describe('Pool', function () {
     });
   });
 
+  it('should not terminate worker if abort listener is defined dedicated worker with Timeout', function () {
+    var workerCount = 0;
+    var pool = createPool(__dirname + '/workers/cleanup-abort.js', {
+      maxWorkers: 1,
+      onCreateWorker: () => {
+        workerCount += 1;
+      }
+    });
+    
+    return pool.exec('asyncTimeout', [],  {
+    })
+    .timeout(200)
```

Comment:
Indentation missing here I think.

(We should set up prettier for this project to automatically format the code 😅 )